src/sentry/static/sentry/app/components/sidebar/sidebarPanel.tsx

```diff
@@ -14,31 +16,54 @@ type Props = React.HTMLProps<HTMLDivElement> & {
   hidePanel?: () => void;
 };
 
-const SidebarPanel = ({
-  orientation,
-  collapsed,
-  hidePanel,
-  title,
-  children,
-  ...props
-}: Props) => (
-  <StyledSidebarPanel collapsed={collapsed} orientation={orientation} {...props}>
-    <SidebarPanelHeader>
-      <Title>{title}</Title>
-      <PanelClose onClick={hidePanel} />
-    </SidebarPanelHeader>
-
-    <SidebarPanelBody>{children}</SidebarPanelBody>
-  </StyledSidebarPanel>
-);
-
-SidebarPanel.propTypes = {
-  title: PropTypes.string,
-  collapsed: PropTypes.bool,
-  orientation: PropTypes.oneOf(['top', 'left']),
-  hidePanel: PropTypes.func,
+/**
+ * Get the container element of the sidebar that react portals into.
+ */
+export const getSidebarPanelContainer = () =>
+  document.getElementById('sidebar-flyout-portal');
+
+const makePortal = () => {
```

Comment:
You could combine these into a `getPortal()` function that creates or gets the portal.

Reply:
I didn't want it to have side effects is all, so I had two